### ISSUE TITLE
Chiseled block not counting for a room

### ISSUE THREAD

**Issue by Saricane** [anegostudios/VintageStory-Issues#1272]
**Game Version:** 1.15.5, survival mode
**worldgen: 1.15.0-pre11
**Platform:** Windows
**Modded:** No 

### Description
Since 1.15, the new rules for defining a room including chiseled blocks are 

> Now count as valid blocks for rooms, with the following limitations: The inward facing side must be almost solid (less than 20 voxels missing), at least 50% of the block's volume must be retained, you still need to use the right materials, e.g. stone for cellars. 

I have a room essentially made of polished white marble.  It actually counts as a room, which is absolutely essential under arctic regions. Having never used block chiseling before (not an artist :-), I made a simple test on one of the blocks on a wall. I did not even carve the block, just "activated" chiseling on this block. Since, this is no more a room. As soon as I replace this block by an unmodified stock block, room recognition is activated again (see hunger rate on the screenshots).

### Screenshots
![2021-08-12_18-57-08](https://user-images.githubusercontent.com/84909583/129240732-c18d76c4-7a9e-4b44-96f4-7c1167d2db9f.png)
![2021-08-12_18-57-25](https://user-images.githubusercontent.com/84909583/129240796-8f55a222-4702-4daa-a476-fea0bf214ac4.png)

**Comment by tyronx** [comment]
any chance you left the chunks, or rejoined the world inbetween though? because that's the issue that if found. Upon unloading that information is lost until the block is updated again by adding/removing voxels

**Comment by Saricane** [comment]
No. I remember I tried chiseling this block (already part of the wall), without any interruption. All in the same session, without moving. And only the next winter I realised I was no more protected, and find the source of the problem by replacing this block by a non chiseled one.

**Comment by tyronx** [comment]
Strange. Well beyond that the room search now also deals with 2 more issues and I also added a debug visualization. If its not fixed next release, at least we'll be able narrow down the source of the issue.

thanks for the info
